### PR TITLE
Skip ruby integration tests

### PR DESCRIPTION
**Pull request by reshmabidikar**
Skip Ruby integration tests

### CODE REVIEW

File: .github/workflows/integration_test.yml [hunk 1]
```diff
@@ -67,11 +67,16 @@ on:
         required: false
         default: true
         type: boolean
-      test-goal:
-        description: The Maven goal(s) used by test jobs
+      test-goal-java:
```
Comment (pierre) [inline .github/workflows/integration_test.yml:70]: Don't forget to update all places where we use this! 😅

**Review by pierre — APPROVED** [review]

File: .github/workflows/integration_test.yml [hunk 2]
```diff
@@ -219,9 +224,9 @@ jobs:
             sleep 1
           done
       - name: Run Java Integration Tests
-        if: inputs.test-goal != '' 
+        if: inputs.test-goal-java != '' 
         run: |
-          ./mvnw --errors --batch-mode --show-version -P${{ matrix.integration-test-profile }} ${{ inputs.maven_args_tests }} ${{ inputs.test-goal }}
+          ./mvnw --errors --batch-mode --show-version -P${{ matrix.integration-test-profile }} ${{ inputs.maven_args_tests }} ${{ inputs.test-goal-java }}
       - name: Checkout killbill-integration-tests
```
Comment (pierre) [inline .github/workflows/integration_test.yml:230]: You can also skip this if `inputs.test-goal-ruby != '' `

Comment (reshmabidikar) [reply .github/workflows/integration_test.yml:230]: Fixed

File: .github/workflows/integration_test.yml (hunk 1 above)
Comment (reshmabidikar) [reply .github/workflows/integration_test.yml:70]: Yup, this is only used in the `invgrp-demo` plugin, I'll update it.

**Comment by reshmabidikar** [comment]
@pierre fixed review comments and made some minor optimizations (See 94fa4cf). If this looks good, I'll merge this PR.

**Pull request merged**